.github/workflows/ci.yml

```diff
@@ -2,29 +2,43 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
   pull_request:
-    branches: [ master ]
+  schedule:
+    - cron: '0 0 * * 5'
+  workflow_dispatch:
```

Comment:
Can you explain the motivation for these changes?

Reply:
The scheduled/CRON event ensures that CI is executed once a week, despite the codebase not being updated. CI did not run in the last 11 days, since the last commit was pushed until I opened a PR. Since edalize has dependencies, it is desirable to run CI periodically in order to catch regressions/breaking changes.

The workflow_dispatch event allows triggering the workflow manually through the web GUI, or through the GitHub API.

Reply:
Can you add a comment to that effect to the file?

Reply:
Done.